sequence_task_3/student/include/impl.h

```diff
@@ -11,4 +12,16 @@
  *
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+std::string reverseWords(const std::string& input) {
+    std::stringstream output;
+    std::string res;
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es22-dont-declare-a-variable-until-you-have-a-value-to-initialize-it-with